src/components/app.tsx

```diff
@@ -112,3 +148,11 @@ export class App extends React.PureComponent<Props, {}> {
     });
   }
 }
+
+export const App = connect<ConnectProps, null, OwnProps>(
+  (state: State) => {
+    return {
+      connectData: selectData(state)
```

Comment:
data and connectData are conflicting -- maybe it's still not a bad idea to split app-root?